components/panel/style.scss

```diff
@@ -129,6 +129,15 @@
 	&:empty {
 		margin-top: 0;
 	}
+	
+	.blocks-base-control {
+		width: 100%;
+	}
+	
+	.blocks-base-control:last-child,
+	.blocks-toggle-control > .blocks-base-control__label {
```

Comment:
I'm concerned how this is tying toggle control to panel components, and am wondering if the desired goal here could be generalized more.

Two thoughts of alternatives:

- Should it be that all `.blocks-base-control__label` within `PanelRow` have their bottom margin removed?
- Should it be that all `ToggleControl` labels have their bottom margin removed? (applied in `toggle-control/style.scss`)

Reply:
The general problem (which #4997 also has) is that .blocks-base-control is forcing a margin at the bottom of each control. There isn't any utility class which removes that margin forcing it to be removed in this instance, where each control is stacked without any space between.

Reply:
Right, but should it be that one or the other of `.blocks-base-control` and `.blocks-base-control__label` applies a bottom margin, not both?

Reply:
Removing the bottom margin from `.blocks-base-control__label` is due to their being no bottom margin on these controls as they exist currently (For these controls I'm changing). They have a `margin-bottom: 5px` when used for block inspector controls, but to not change how they look in this case I removed the margin.

It wouldn't make sense to remove that margin for all controls, as having some margin between the label and input/textarea controls makes sense. 

Maybe opting to globally remove the label margin for toggle controls would be the best way forward?

<img width="279" alt="screen shot 2018-02-22 at 12 19 31" src="https://user-images.githubusercontent.com/704594/36538289-c0090da8-17ca-11e8-896d-e462d232e8fe.png">

Reply:
I think applying globally `.blocks-base-control:last-child { margin-bottom: 0 } ` would make sense, as currently the last control in a series of controls does add some updates unnecessary whitespace.

<img width="296" alt="screen shot 2018-02-22 at 12 32 47" src="https://user-images.githubusercontent.com/704594/36538820-b098217c-17cc-11e8-807b-28f62fa70613.png">